### DISCUSSION TITLE
Javadoc on the site refers to a removed custom domain

### DISCUSSION THREAD

**Discussion by leduyquang753** [community/community#21932] [Other Feature Feedback, Questions, & Ideas]
<p>Hi everyone. I have made a <a href="https://leduyquang753.github.io" title="My Website!" rel="nofollow noopener">website</a> for my GitHub account, and on the site, I also have placed my new ClickItAPI javadoc on another place (<a href="https://leduyquang753.github.io/ClickItAPI" title="The javadoc" rel="nofollow noopener">here</a>), and I assigned a custom domain to the site. But it didn’t work properly, so I removed it, and then when I reentered the javadoc, it still redirected to the old domain and the page says can’t find DNS address of the server with my old domain (I cancelled it).</p>
<p>The main page still works fine and the same as the javadoc on my computer.</p>
<p>Can you give me some guests?</p>
<p><strong>SOLVED: </strong> I cleared my cache and it worked. Oh my…</p>

**Comment by birdcar** [comment, marked as answer]
<p>Glad you were able to resolve the issue! Cache issues always trip me up when I least expect it. </p>
<p>Thanks for visiting the GitHub Community.</p>
